cfgov/v1/migrations/0232_set_filter_children_on_newsroom_activitylog.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 2.2.16 on 2020-09-24 19:20
+
+from django.db import migrations
+
+from v1.util.migrations import migrate_page_types_and_fields
+
+
+def forward_mapper(page_or_revision, data):
+    # Manipulate the stream block data forwards
+    data['filter_children'] = False
+    return data
+
+
+def forwards(apps, schema_editor):
+    page_types_and_fields = [
+        ('v1', 'ActivityLogPage', 'content', 'filter_controls'),
+        ('v1', 'NewsroomLandingPage', 'content', 'filter_controls'),
```

Comment:
Newsroom no longer includes Blog entries, so I think this is incorrect? We do want it to filter its children.
```suggestion
```

Reply:
@Scotchester In the current code, `NewsroomLandingPage.filterable_children_only = False`, meaning that Newsroom filters any pages that meet its (category) criteria, not just its children. Because this PR defaults `FilterableList.filter_children = True` (filter only children by default), I think we do need to set this line to maintain the current behavior.

I believe Newsroom includes any pages in [one of the newsroom categories](https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/v1/util/ref.py#L100-L105), using the logic [you ask about down below](https://github.com/cfpb/consumerfinance.gov/pull/6039/files#r495289839).

Are you suggesting that these two things happen to be equivalent in the current site:

- Filterable children of Newsroom
- Filterable pages in the newsroom categories, anywhere on the site

I did a query and there appear to be 6 live filterable pages marked with newsroom categories that don't live under the Newsroom:

- [Truth in Savings Act (TISA) examination procedures](https://www.consumerfinance.gov/policy-compliance/guidance/supervision-examinations/truth-in-savings-act-tisa-examination-procedures/)
- [Fair Debt Collection Practices Act (FDCPA) examination procedures](https://www.consumerfinance.gov/policy-compliance/guidance/supervision-examinations/fair-debt-collection-practices-act-fdcpa-examination-procedures/ )
- [Fair Credit Reporting Act (FCRA) examination procedures](https://www.consumerfinance.gov/policy-compliance/guidance/supervision-examinations/fair-credit-reporting-act-fcra-examination-procedures/)
- [Consumer Leasing Act (CLA) examination procedures](https://www.consumerfinance.gov/policy-compliance/guidance/supervision-examinations/consumer-leasing-act-cla-examination-procedures/)
- [Student Debt Domino Effect?](http://www.consumerfinance.gov:8000/about-us/blog/student-debt-domino-effect/)
- [Help for Military Homeowners](https://www.consumerfinance.gov/about-us/blog/help-for-military-homeowners/)

You can confirm this by e.g. searching the newsroom for ["domino"](https://www.consumerfinance.gov/about-us/newsroom/?title=domino&from_date=&to_date=) and seeing that a blog page is returned. Whether or not this is intended behavior I don't know, but it is current behavior.

Reply:
Hmm… I see. I think it is probably _not_ intended behavior, but I guess better safe than sorry, so let's leave it that way for now.